vllm/transformers_utils/config.py

```diff
@@ -87,6 +88,7 @@ def _get_hf_token() -> Optional[str]:
     "medusa": MedusaConfig,
     "eagle": EAGLEConfig,
     "exaone": ExaoneConfig,
+    "exaone4": Exaone4Config,
```

Comment:
I have another question. 
Is `Exaone4Config` necessary? or can we just use the default Exaone4 HF config?